source/vistas/cli/startup_project.py

```diff
@@ -0,0 +1,12 @@
+import click
+from vistas.cli import cli
+from vistas.ui.app import App
+import os
+
+
+@cli.command(short_help='Start VISTAS with a predefined project loaded.')
+@click.argument('project', type=click.Path(exists=True))
+def startup_project(project):
+    os.chdir(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))  # Todo - macOS equivalent?
```

Comment:
Please add a comment explaining this. Also, this could be cleaned up a bit:

```python
source_dir = os.path.abspath(__file__)
for _ in range(3):  # Comment about why here
    source_dir = os.path.dirname(source_dir)
os.chdir(source_dir)
```